Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/POMLearnConfigWizardPage.xaml.cs

```diff
@@ -87,17 +87,30 @@ public void WizardEvent(WizardEventArgs WizardEventArgs)
                     if (mWizard.POM.TargetApplicationKey != null)
                         mAppPlatform =  WorkSpace.UserProfile.Solution.GetTargetApplicationPlatform(mWizard.POM.TargetApplicationKey);
                     mWizard.OptionalAgentsList = GingerCore.General.ConvertListToObservableList((from x in WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<Agent>() where x.Platform == mAppPlatform select x).ToList());
-                    foreach (Agent agent in mWizard.OptionalAgentsList)
+                    foreach (Agent agent in GingerCore.General.ConvertListToObservableList((from x in WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<Agent>() where x.Platform == mAppPlatform select x).ToList()))
                     {
                         agent.Tag = string.Empty;
+
+                        // Filter the Agents List and remove Headless Browsers for POM Learning..
+                        if(agent.DriverType == eDriverType.SeleniumChrome || agent.DriverType == eDriverType.SeleniumFireFox)
+                        {
+                            var selDriverObj = agent.DriverConfiguration as ObservableList<DriverConfigParam>;
+                            DriverConfigParam drivConfigParam = selDriverObj.Where(x => x.ItemName == "HeadlessBrowserMode").FirstOrDefault();
+                            if (drivConfigParam != null && drivConfigParam.Value.ToLower().Equals("true"))
+                            {
+                                mWizard.OptionalAgentsList.Remove(agent);
+                            }
+                        }
                     }
```

Comment:
let's remove the selenuimDriver hidden browser filtering,
i don't want to mess this generic wizrd which will support all Agents in future with multi IF's checking specific driver configs in it.
user can understand using hidden browser won't work here.